docs/csharp/programming-guide/concepts/linq/linq-to-xml-overview.md

```diff
@@ -24,26 +24,43 @@ XML has been widely adopted as a way to format data in many contexts. For exampl
   
  For example, you might have a typical XML purchase order as described in [Sample XML File: Typical Purchase Order (LINQ to XML)](sample-xml-file-typical-purchase-order-linq-to-xml-1.md). By using [!INCLUDE[sqltecxlinq](~/includes/sqltecxlinq-md.md)], you could run the following query to obtain the part number attribute value for every item element in the purchase order:  
   
-```csharp  
-IEnumerable<string> partNos =  
-from item in purchaseOrder.Descendants("Item")  
-select (string) item.Attribute("PartNumber");  
-```  
-  
+```csharp
+// Load the XML file containing the purchase orders
+string folderPath = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
+XElement purchaseOrder = XElement.Load($"{folderPath}\\PurchaseOrder.xml");
+
+IEnumerable<string> partNos =  from item in purchaseOrder.Descendants("Item")  
+                               select (string) item.Attribute("PartNumber");  
+``` 
+This can be rewritten in method syntax form:
+
+```csharp
+IEnumerable<string> partNos = purchaseOrder.Descendants("Item").Select(x => x.Attribute("PartNumber"));
```

Comment:
You should add a cast or `ToString` method so the type of the `Select` matches.

Reply:
An easy fix, I'll add it asap.